frontend/webpack.config.ts

```diff
@@ -68,6 +70,12 @@ const config: Configuration = {
               workers: require('os').cpus().length - 1,
             },
           },
+          isDevelopment && {
+            loader: 'babel-loader',
+            options: {
+              plugins: ['react-refresh/babel'],
+            },
+          },
```

Comment:
I'd suggest this instead, which is more clear to me than `filter(Boolean)`

```suggestion
          ...(isDevelopment ? [{
            loader: 'babel-loader',
            options: {
              plugins: ['react-refresh/babel'],
            },
          }] : []),
```

Reply:
@spadgett 
IMO its way less readable than `.filter(Boolean)`.
can we up this for debate? or you decide :)

Reply:
btw this is the "accepted" way to do this in the web

Reply:
I disagree:

1. This is an idiom we use in many places in the console code today
2. The `filter` call is in a different place in the code. It's unclear how or if these two things are related, and the `filter` call is confusing without that context.
3. It seems hacky to knowingly insert invalid values into the array and remove them later

Reply:
You don't necessarily need to take my suggestion, but imo we should find a different way than `.filter(Boolean)`